blackbox/docs/best_practice/bootstrap_checks.txt

```diff
@@ -0,0 +1,67 @@
+.. highlight:: sh
+
+.. _bootstrap_checks:
+
+================
+Bootstrap Checks
+================
+
+If you are binding to a network reachable IP address, CrateDB performs a number
+of bootstrap checks during startup. These checks examine your setup and will
+prevent startup if a problem is detected.
+
+This best practices guide is intended to help you configure your setup so that
+CrateDB passes the bootstrap checks and can perform optimally.
+
+If you are binding to the loopback address, the bootstrap checks will not be
+run, but it is still a good idea to follow these instructions.
+
+System Settings
+===============
+
+Official Packages
+-----------------
+
+If you are using one of the official packages, all of the necessary operating
+system configuration is handled for you.
+
+Tarball
+-------
+
+If you have installed CrateDB from a tarball, you must manually configure a
+number of operating system settings.
+
+Edit ``/etc/security/limits.conf`` and configure these lines::
```

Comment:
this is only is true for linux. i think you need to describe "what" needs to be done. and if you describe "how" then it should be OS specific. at least osx is different.